rust/examples-frontend/echo/index.html

```diff
@@ -18,14 +18,14 @@
     }
 </style>
 <div id="container">
-    <figure>
-        <video id="preview-send" muted controls></video>
-        <figcaption>Send preview</figcaption>
-    </figure>
-    <figure>
-        <video id="preview-receive" muted controls></video>
-        <figcaption>Receive preview</figcaption>
-    </figure>
-</div>
+        <figure>
```

Comment:
dunno how to run lint

Reply:
Why did you modify it at all?

This project is too simple to have HTML linter

Reply:
I touched that file recently
then copy-pasted this file from master and here is result ...